packages/picasso.js/src/core/scales/color/categorical.js

```diff
@@ -36,8 +66,10 @@ export default function scaleCategorical(settings = {}, data = {}, resources = {
   const theme = resources.theme;
   const stgns = resolveSettings(settings, DEFAULT_SETTINGS, { data, resources });
   stgns.explicit = resolveSettings(settings.explicit, DEFAULT_EXPLICIT_SETTINGS, { data, resources });
+  const haveOverride = settings.explicit && settings.explicit.override;
```

Comment:
add `override: false` to `DEFAULT_EXPLICIT_SETTINGS` and use `stngs.explicit.override`